### PR TITLE
config: Add content to the "Introduction" page

### PR DESCRIPTION
**Pull request by teodutu**
# Prerequisite Checklist

- [x] Read the [contribution guidelines](https://github.com/cs-pub-ro/operating-systems/blob/main/CONTRIBUTING.md#pull-requests) regarding submitting new changes to the project;
- [x] Tested your changes against relevant architectures and platforms;
- [x] Updated relevant documentation (if needed).

## Description of changes

It was a template page with no relevant information. Now it provides short descriptions for each chapter.

### CODE REVIEW

File: landing-page/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Operating Systems
+
+Welcome to the Operating Systems course, held by the National University of Science and Technology Politehnica Bucharest.
+
+Our content is structured into 5 chapters:
+
+- **Software Stack:** provides an overview about the interactions between applications, libraries, the and the hardware with an emphasis on the need and usage for system calls
+- **Data:** teaches how the operating system manages the memory of a process
+- **Compute:** explains how the operating system manages threads, processes and the way they interact
+- **I/O:** teaches how an  application interacts with outside resource such as files or the network from the perspective of the operating system
```
Comment (fabianpatras) [inline landing-page/README.md:14]:
```suggestion
- **I/O:** teaches how an application interacts with outside resource such as files or the network from the perspective of the operating system
```

**Review by fabianpatras — COMMENTED** [review]
Other than that small change, should we have here some contact information / a pointer to github issues where to submit problems / questions / fixes (PRs)? (I don't know if we have this somewhere else)

**Review by Alex-deVis — APPROVED** [review]

**Comment by teodutu** [comment]
> Other than that small change, should we have here some contact information / a pointer to github issues where to submit problems / questions / fixes (PRs)? (I don't know if we have this somewhere else)

Done @fabianpatras.

**Review by fabianpatras — APPROVED** [review]

**Review by mariasfiraiala — APPROVED** [review]
:+1:

**Pull request merged**